.github/actions/unit-tests/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Run unit test
+inputs:
+  package-name:
+    description: 'Name of the package'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v3
+      with:
+        node-version-file: .nvmrc
+    - uses: ./.github/actions/setup
+
+    # Find current PR's number
+    - uses: jwalton/gh-find-current-pr@v1
+      id: findPr
+
+    - name: Run unit tests
+      uses: ArtiomTr/jest-coverage-report-action@v2
+      with:
+        working-directory: ./packages/${{ inputs.package-name }}
+        test-script: yarn workspace @editorjs/${{ inputs.package-name }} test
```

Comment:
I'm not sure that it folder name always equals package name.

For example, in playground they are different:

```
/packages/playground
```

```
@editorjs/document-playground
```

Reply:
would variable's name "package-folder-name" be appropriate?

Reply:
Agree with that. They might be different. Also, I think it's better to pass full package name like `@editorjs/document-playground`.

Reply:
Then we would need 2 different vars, like "package-folder-name" and "package-name", cause in some places you need the folder's name without "@editorjs". Is it worth it? 
Or it doesn't always come with @editorjs prefix?

Reply:
The `./packages/` path could be also unhardcoded I guess

Reply:
>in some places you need the folder's name without "@editorjs"

it is not a folder, but a package name, so it alway will include `@editorjs/...`

Reply:
You need 2 vars, like `working-directory` and `package-name`

Reply:
So, have 2 inputs like:
`package-path` (./package/name_of_your_package_folder)
and
`package-name`? (@editorjs/name_of_your_package)